src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  if (sourceString.length === 0) {
+    return styles;
+  }
+
+  sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el.includes(':'))
+    .forEach((el) => {
```

Comment:
- arr/array/str/string/el/elem/obj/object/res/result are bad names, name should describe variable
- You can use `reduce` to generate an object with it